website/docs/Examples/Integrate - OpenAI.md

```diff
@@ -0,0 +1,183 @@
+FLAML has integrated the OpenAI's completion API.
+### Prerequisites
+
+Install the [openai] option.
```

Comment:
Please specify the required version number for flaml (1.1.3). And also mention that this feature is subject to change in future.